src/lib/utils/roller.ts

```diff
@@ -73,19 +72,19 @@ export const spawn = async (
 };
 
 export const configureKeys = async (
-  api: any,
-  _wallet: any,
-  _point: number,
+  api: RollerRPCAPI,
+  wallet: any,
+  point: Ship,
   proxy: string,
   nonce: number,
-  urbitWallet: any
+  networkSeed: any,
```

Comment:
Is there a better choice than `any` in this case?

Reply:
I think we can use string here, from the `urbit-key-generation` repo [this](https://github.com/urbit/urbit-key-generation/blob/fa656d1d982681ea10c6647ec0b878e0b9309c07/src/index.js#L273) should be a hex-encoded string

Reply:
...although this is a Maybe (Just/Nothing) from folktale, and that doesn't seem to support typescript? https://github.com/origamitower/folktale/issues/65